src/operator/LimitsLabel.tsx

```diff
@@ -0,0 +1,14 @@
+import React, {FC} from 'react'
+import {get} from 'lodash'
+import {OrgLimits} from 'src/types'
+
+interface Props {
+  name: string
+  limits: OrgLimits
+}
+
+const LimitsLabel: FC<Props> = ({name, limits}) => {
+  return <p className="operator-limits-label">{get(limits, name, '')}</p>
```

Comment:
I like the direction of abstracting this into a separate component, but it seems a bit unnecessary. Part of the abstraction for this was so that we could explicitly set the type, but that looks like it's being unused here. We might as well reduce our usage of lodash's `get` by just inlining these and not have it be a component that does 1 thing